tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -161,6 +162,40 @@ def user_adds_publication_role_to_user_via_api(
         f'Adding publication role to user API request failed with {response.status_code} and {response.text}'
 
 
+def user_removes_roles_from_prerelease_user_via_api(user_id: str) -> None:
+    response = admin_client.delete(
+        f'/api/user-management/user/{user_id}/release-role/pre-release'
```

Comment:
Given that this endpoint specifically mentions `release-role`, I'd not expect it to delete any UserPublicationRoles from the User - going back to my other comments on the `RemoveUserPreReleaseRoles` method though, I think we'd be removing that behaviour anyway.

Reply:
I've renamed this to `user_removes_all_release_and_publication_roles_from_user` now